lib/fortuneteller/job/day_plan.rb

```diff
@@ -21,23 +24,12 @@ def add_savings_plan(percent:, match:, account:, holding:)
         }
       end
 
-      def adjusted_base(year, growth_rates)
```

Comment:
Hah - should have looked at all commits at once instead of in order.  Looks like you did end up moving a lot of this calculation to InflatingInt.  I think the point about using `sim.inflating_int` instead of creating one directly still stands.